extended-statistics.md

```diff
@@ -0,0 +1,157 @@
+---
+title: Introduction to Extended Statistics
+summary: Learn how to use extended statistics to guide the optimizer.
+---
+
+# Introduction to Extended Statistics
+
+TiDB can collect the following two types of statistics:
+
+- Regular statistics: statistics such as histograms and Count-Min Sketch. See [Introduction to Statistics](/statistics.md) for details.
+- Extended statistics: statistics filtered by tables and columns.
+
+When the `ANALYZE` statement is executed manually or automatically, TiDB by default only collects the regular statistics and does not collect the extended statistics. This is because the extended statistics are only used for optimizer estimates in specific scenarios, and collecting them requires additional overhead.
+
+Extended statistics are disabled by default. To collect extended statistics, you need to first enable and register the extended statistics.
```

Comment:
better to separate the description of enabling and registration of extended statistics.  Enabling is an overall switch, and one-time task. Registration is for individual extended statistics, and you need repeat the registration for each extended statistics.

Reply:
done